CONTRIBUTING.md

```diff
@@ -33,29 +33,35 @@ cargo clippy --workspace --tests -- --deny warnings
 
 Execute `cargo fmt --all` to fix the formatting.
 
-## Generating Docs
-
-If you make changes to [Roc's Standard Library](https://www.roc-lang.org/builtins/Str), you can add comments to the code following [the CommonMark Spec](https://spec.commonmark.org/current/) to further explain your intentions. You can view these changes locally with:
-
-```sh
-cargo run docs crates/compiler/builtins/roc/main.roc
-```
-
-This command will generate the documentation in the [`generated-docs`](generated-docs) directory.
-
 ## Contribution Tips
 
 - If you've never made a pull request on github before, [this](https://www.freecodecamp.org/news/how-to-make-your-first-pull-request-on-github-3/) will be a good place to start.
-- Create an issue if the purpose of a struct/field/type/function/... is not immediately clear from its name or nearby comments.
 - You can find good first issues [here][good-first-issues]. Once you have gained some experience you can take a look at the [intermediate issues](https://github.com/roc-lang/roc/issues?q=is%3Aopen+is%3Aissue+label%3A%22intermediate+issue%22).
-- [Fork](https://github.com/roc-lang/roc/fork) the repo so that you can apply your changes first on your own copy of the roc repo.
```

Comment:
You have no choice but to fork the repo, forking is also described in the first PR on github link.

Reply:
For a completely new contributor to OSS that may not be obvious. It wasnt for me at the start.

Reply:
Yes, that is probably a more helpful resource. I could have been clearer with my comment, it's ok to remove this. It's good tou have already thought about the new users.